client/src/infographic/Infographic.js

```diff
@@ -188,6 +195,33 @@ class InfoGraph_ extends React.Component {
         include_tables: false,
       }}
     />;
+    const filter_icon_props = {
+      color: window.infobase_color_constants.primaryColor,
+      title: text_maker("filter_panels"),
+      width: 15,
+      height: 15,
+      vertical_align: 5,
+    };
+    const is_filter_panels = active_bubble_id === "financial" || active_bubble_id === "people";
+    const panel_renderers = !loading &&
+      _.chain(panel_keys)
+        .map((panel_key) => {
+          const panel_obj = PanelRegistry.lookup(panel_key, subject.level);
+          const active_panel_filter_keys = _.chain(panel_filter)
+            .map((value, key) => value && key)
+            .compact()
+            .value();
+          const is_filtered = _.intersection(active_panel_filter_keys, panel_obj.depends_on).length > 0 || !is_filter_panels;
```

Comment:
Not sure if `is_filtered` and `!is_filter_panels` naming seem make sense?

Reply:
That was part of what had me scratching my head earlier. I think right now having a value of `true` in `panel_filter` means a key is NOT being filtered out, i.e. is being included in results. The name `panel_filter` kind of makes it feel like it should be the other way around.

The text associated with this feature on the site is "Panel Filters", which I think is a good user-facing name for it, but internally `panel_filter`, the component state, is more like a `data_sources_to_display` object. Maybe that changes/is complicated if we add more filter options on the site, but I think it's a much easier name to use/think in terms of for now.